### ISSUE TITLE
Glade-3.18.3

### ISSUE THREAD

**Issue by dummys** [Homebrew/homebrew#43982]
Hi,

Is it possible to update glade to add the galde 3.18.3 which use gtk+3 and not gtk+ ?
Thanks

**Comment by dunn** [comment]
We'd love a pull request! Take a look at https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md and let us know if you have questions :ok_woman:

**Comment by tschoonj** [comment]
I have also looked into this in the past but had to give up since this version needs a lot of patching before it can be compiled with gtk-quartz backend and gtk-mac-integration. The 3.18.3 version was unfortunately never tested properly on a Mac before it was released.

The current master branch compiles nicely so the issues have been fixed upstream. Hopefully they will make a release soon, perhaps already next week when gnome 3.18 will be launched.
